doc/prompt_task_map.json

```diff
@@ -0,0 +1,746 @@
+            ]
+        },
+        {
+            "task": "task580_socialiqa_answer_generation",
+            "prompt": [
+				"Show choices and generate answer",
+				"Show choices and generate index"
+            ]
+        }
+    ],
+    "wiqa": [
+    ],
+    "cosmos_qa": [
+        {
+            "task": "task024_cosmosqa_answer_generation",
+            "prompt": [
+				"context_description_question_text",
+				"context_question_description_text",
+				"description_context_question_text"
+            ]
+        }
+    ],
+    "qasc": [
+    ],
+    "quarel": [
+        {
+            "task": "task1378_quarel_correct_answer_generation",
+            "prompt": [
+				"choose_between",
+				"do_not_use",
+				"heres_a_story",
+				"logic_test",
+				"testing_students"
+            ]
+        }
+    ],
+    "sciq": [
+        {
+            "task": "task591_sciq_answer_generation",
+            "prompt": [
+				"Direct Question (Closed Book)"
+            ]
+        }
+    ],
+    "wiki_hop/original": [
+    ],
+    "wiki_hop/masked": [
+    ],
+    "adversarial_qa/adversarialQA": [
+    ],
+    "adversarial_qa/dbidaf": [
+    ],
+    "adversarial_qa/dbert": [
+    ],	
+    "adversarial_qa/droberta": [
+    ],
+    "quoref": [
+        {
+            "task": "task002_quoref_answer_generation",
+            "prompt": [
+				"Answer Friend Question",
+				"Answer Question Given Context",
+				"Answer Test",
+				"Context Contains Answer",
+				"Find Answer",
+				"Found Context Online",
+				"Given Context Answer Question",
+				"Guess Answer",
+				"Read And Extract",
+				"What Is The Answer"
+            ]
+        }
+    ],
+    "duorc/SelfRC": [
+        {
+            "task": "task193_duorc_question_generation",
+            "prompt": [
+				"generate_question"
+            ]
+        },
+        {
+            "task": "task194_duorc_answer_generation",
+            "prompt": [
+				"answer_question",
+				"decide_worth_it",
+				"question_answering",
+				"movie_director",
+				"extract_answer"
+            ]
+        }
+    ],
+    "duorc/ParaphraseRC": [
+        {
+            "task": "task193_duorc_question_generation",
+            "prompt": [
+				"generate_question"
+            ]
+        },
+        {
+            "task": "task194_duorc_answer_generation",
+            "prompt": [
+				"answer_question",
+				"decide_worth_it",
+				"question_answering",
+				"movie_director",
+				"extract_answer"
+            ]
+        }
+    ],
+    "ropes": [
+        {
+            "task": "task061_ropes_answer_generation",
+            "prompt": [
+				"background_new_situation_answer",
+				"background_situation_middle",
+				"given_background_situation",
+				"new_situation_background_answer",
+				"plain_background_situation",
+				"plain_bottom_hint",
+				"prompt_beginning",
+				"prompt_bottom_hint_beginning",
+				"prompt_mix",
+				"read_background_situation"
+            ]
+        }
+    ],
+    "hotpot_qa/distractor": [
```

Comment:
T0 casts hotpot_qa as a closed_book qa task. Can we add a `hotpot_qa/closed_book` placeholder here, and add the task together with other missing tasks?

Reply:
Can you elaborate more?
They don't have any closed_book prompt. are you suggesting adding this task without prompt?

Reply:
It seems they used the [kilt version of hotpotqa](https://huggingface.co/datasets/kilt_tasks/viewer/hotpotqa/train), according the the list [here](https://github.com/bigscience-workshop/t-zero/blob/2604dd383c72e864b73efb3c5bce19892c7a679c/t0/datasets.csv).

Reply:
Thanks! I'll add it.